toolkits/springernature/packages/springernature-header/README.md

```diff
@@ -0,0 +1,69 @@
+# Springer Nature User Details
+
+Springer nature branded header component, with Springer Nature Editorial logo, optional journal logo, and the user-details component.
+
+## Requirements
+
+A class name of `.js` on the document root element will be required to enable visual styles for the user info drop down. The class should be added via a micro (but blocking) script, placed as far up in the <head> of the document as possible.
+
+```javascript
+<script>
+    // JS Detection Script
+    (function(e){var t=e.documentElement,n=e.implementation;t.className='js';})(document)
+</script>
+```
+
+## Usage
+
+### HTML
+
+The component assumes you have the following data available to your template:
+
+logoutUri
+showJournalLogo - boolean to indicate whether or not there is a journal logo
+backUrl - the href for the anchor wrapping the header logos (usually to the home page)
+user-details {
+    first_name
+    last_name
+    email_address
+}
+submission {
+    journalId
+    journalName
+}
+
+It requires the springernature-user-details component. You will need to import the global context and the user details components.
```

Comment:
link to the `springernature-user-details` component, and it won't be the global context. As this is springernature it will be the springernature brand of the `brand-context`

Reply:
I would also position the use of `user-details` as optional as it relies on including another component, something like... "if the `user-details` data is included then it requires the inclusion of the user-details component (link)"

Reply:
I have mentioned the global context as this component is only using the breakpoints mixins from the default context.